cmd/api/src/database/audit.go

```diff
@@ -40,13 +40,18 @@ func newAuditLog(context context.Context, entry model.AuditEntry, idResolver aut
 
 	auditLog := model.AuditLog{
 		Action:          entry.Action,
-		Fields:          types.JSONUntypedObject(entry.Model.AuditData()),
 		RequestID:       bheCtx.RequestID,
 		SourceIpAddress: bheCtx.RequestIP,
 		Status:          entry.Status,
 		CommitID:        entry.CommitID,
 	}
 
+	if entry.Model != nil {
+		auditLog.Fields = types.JSONUntypedObject(entry.Model.AuditData())
+	} else {
+		auditLog.Fields = types.JSONUntypedObject{}
+	}
+
```

Comment:
This adds panic protection in the event that newAuditLog is called without an entry.Model intialized